src/test/resources/META-INF/mods.toml

```diff
@@ -65,3 +65,11 @@ loaderVersion="[28,)"
     modId="stencil_enable_test"
 [[mods]]
     modId="deferred_registry_test"
+[[mods]]
+    modId="ender_mask_test"
+[[dependencies.global_loot_test]]
```

Comment:
I am not sure this makes sense here.
Should that not be dependencies.ender_mask_test?

Reply:
It's fix